### PR TITLE
feat: rework navigation, update breadcrumbs

### PR DESCRIPTION
**Pull request by artemmufazalov**
Breadcrumbs are not ready yet, it's an intermediary version

### CODE REVIEW

File: src/containers/AsideNavigation/AsideNavigation.tsx [hunk 1]
```diff
@@ -115,46 +111,8 @@ interface AsideNavigationProps {
...
-        icon: clusterIcon,
-        iconSize: 20,
-        location: createHref(routes.cluster, {
-            activeTab: CLUSTER_PAGES.cluster.id,
-        }),
-        locationKeys: ['/cluster/cluster'],
-    },
-];
+// FIXME: add items or delete
+const items: MenuItem[] = [];
```
Comment (artemmufazalov) [inline src/containers/AsideNavigation/AsideNavigation.tsx:115]: I left it as it was to keep logic and to fill it with new items on the next iteration

File: src/containers/Tenants/Tenants.js [hunk 2]
```diff
@@ -94,14 +86,15 @@ class Tenants extends React.Component {
...
             <div className={b('controls')}>
-                <TextInput
-                    className={b('search')}
-                    placeholder="Database name"
-                    value={searchQuery}
-                    onUpdate={handleSearchQuery}
-                    hasClear
-                    autoFocus
-                />
+                <div className={b('search')}>
```
Comment (artemmufazalov) [inline src/containers/Tenants/Tenants.js:89]: `TextInput` classname sometimes incorrectly applied, so I wrapped it with a `div` with `classname` instead

File: src/containers/Versions/Versions.tsx [hunk 3]
```diff
@@ -55,67 +64,70 @@ export const Versions = ({nodes = [], versionToColor}: VersionsProps) => {
...
-    const renderGroupedNodes = () => {
-        const tenantNodes = getGroupedTenantNodes(nodes, versionToColor, groupByValue);
-        const storageNodes = getGroupedStorageNodes(nodes, versionToColor);
-        const otherNodes = getOtherNodes(nodes, versionToColor);
-        const storageNodesContent = storageNodes?.length ? (
-            <>
-                <h3>Storage nodes</h3>
-                {storageNodes.map(({title, nodes: itemNodes, items, versionColor}, index) => (
+
+    if (loading && !wasLoaded) {
```
Comment (artemmufazalov) [inline src/containers/Versions/Versions.tsx:68]: Here I removed `renderGroupedNodes` function and added loader. GH considers its as a new code :(

File: src/containers/Cluster/ClusterInfo/ClusterInfo.tsx [hunk 4]
```diff
@@ -204,20 +145,45 @@ export const ClusterInfo = ({
...
+        return (
+            <EntityStatus
+                size="m"
+                status={cluster?.Overall}
+                name={cluster?.Name ?? 'Unknown cluster'}
+                className={b('title')}
+            />
+        );
+    };
+
```
Comment (artemmufazalov) [inline src/containers/Cluster/ClusterInfo/ClusterInfo.tsx:174]: Two different skeletons for the info and for the title to include them in `ClusterInfo` layout and to enable expand button for the skeleton as well. This should enable user to hide content even if it's not loaded yet. May be helpful for big clusters, where `ClusterInfo` could load 20+ seconds, but there could be a need to to view tabs content ASAP

File: src/containers/Cluster/Cluster.tsx [hunk 5]
```diff
@@ -1,59 +1,175 @@
...
+        <div className={b()}>
+            <ClusterInfo
+                cluster={cluster}
+                versionsValues={versionsValues}
+                loading={infoLoading}
+                error={clusterError}
+                additionalClusterProps={additionalClusterProps}
+            />
+
+            <div>
```
Comment (Raubzeug) [inline src/containers/Cluster/Cluster.tsx:148]: What do you think about moving tabs into separate component?

**Review by Raubzeug — CHANGES_REQUESTED** [review]

File: src/containers/Cluster/Cluster.tsx (hunk 5 above)
Comment (artemmufazalov) [reply src/containers/Cluster/Cluster.tsx:148]: I tried and not very liked it. It doesn't help much, but become complicated when we need to customise it (e.g. in Diagnostics there is autorefresh switch that is added to `Tabs`). So, in my view, it doesn't worth it

**Review by Raubzeug — APPROVED** [review]

File: src/containers/Versions/Versions.tsx [hunk 6]
```diff
@@ -55,67 +64,70 @@ export const Versions = ({nodes = [], versionToColor}: VersionsProps) => {
...
+            )}
+        </>
+    ) : null;
+    const otherNodesContent = otherNodes?.length ? (
+        <>
+            <h3>Other nodes</h3>
+            {otherNodes.map(
+                ({title, nodes: itemNodes, items, versionColor, versionsValues}, index) => (
+                    <GroupedNodesTree
+                        key={`other-nodes-${index}`}
```
[the remark this answers is not in the document]
Comment (artemmufazalov) [reply src/containers/Versions/Versions.tsx]: Changed to `title`. Titles are either DBs names, or versions, both should be unique in such view

**Pull request merged**